### PR TITLE
ign -> gz CMake, Python, Partial Source, and File Migrations : gz-physics

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/765

### CODE REVIEW

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

File: tutorials/06-physics-simulation-concepts.md [hunk 1]
```diff
@@ -209,13 +209,13 @@ Twist message to command the `X3` multicopter
 ascending 0.1 m.s as follow:
 
 ```bash
-ign topic -t "/X3/gazebo/command/twist" -m gz.msgs.Twist -p "linear: {x:0 y: 0 z: 0.1} angular {z: 0}"
+gz topic -t "/X3/sim/command/twist" -m gz.msgs.Twist -p "linear: {x:0 y: 0 z: 0.1} angular {z: 0}"
```
[the remark this answers is not in the document]
Comment (methylDragon) [reply tutorials/06-physics-simulation-concepts.md]: Fixed!

File: tutorials/06-physics-simulation-concepts.md [hunk 2]
```diff
@@ -209,13 +209,13 @@ Twist message to command the `X3` multicopter
...
 ```
 
 then hovering:
 
 ```bash
-ign topic -t "/X3/gazebo/command/twist" -m gz.msgs.Twist -p " "
+gz topic -t "/X3/sim/command/twist" -m gz.msgs.Twist -p " "
```
[the remark this answers is not in the document]
Comment (methylDragon) [reply tutorials/06-physics-simulation-concepts.md]: Fixed!

**Review by chapulina — APPROVED** [review]

**Pull request merged**